client-tez/pom.xml

```diff
@@ -176,4 +181,4 @@
             </plugin>
         </plugins>
     </build>
-</project>
+</project>
```

Comment:
We should add a new line, otherwise there will be a warning.

Reply:
ok, we add added it